src/ToolServiceProvider.php

```diff
@@ -26,8 +26,8 @@ public function boot()
             $this->routes();
         });
 
-        Gate::policy(config('permission.models.permission'), PermissionPolicy::class);
-        Gate::policy(config('permission.models.role'), RolePolicy::class);
+        Gate::policy(config('permission.models.permission'), config('permission.policy.permission'));
```

Comment:
Seems like a not good idea to inject custom properties in Spatie's configuration file. I would suggest this approach:

```
 \Vyuldashev\NovaPermission\NovaPermissionTool::make()
            ->rolePolicy(RolePolicy::class)
            ->permissionPolicy(PermissionPolicy::class),
```

`Vyuldashev\NovaPermission\RolePolicy` and `Vyuldashev\NovaPermission\PermissionPolicy` should be used by default.